packages/jss/package.json

```diff
@@ -38,6 +38,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.3.1",
+    "csstype": "^2.6.5",
```

Comment:
did you really mean it as a direct dependency?

Reply:
Yes, most of the time, type dependency should be direct ones. Only in individual cases they should  a peer one (React for example)